fission-web-server/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  postgres:
+    image: postgres:11-alpine
+    env_file: .env
+    ports:
+      - 5432:5432
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+  dns-auth:
+    image: powerdns/pdns-auth-44
+    command: --local-address=0.0.0.0 --local-port=5300
+    user: root
+    env_file: .env
+    ports:
+      - "5300:5300"
+      - "5300:5300/udp"
+      - "8081:8081"
+    networks:
+      default:
+        ipv4_address: 172.25.0.6
+  dns-recursor:
+    image: powerdns/pdns-recursor-44
+    command: --forward-zones=${BASE_DOMAIN}=172.25.0.6:5300,${USER_DOMAIN}=172.25.0.6:5300,${APP_DOMAIN}=172.25.0.6:5300 --forward-zones-recurse=.=1.1.1.1 --local-address=0.0.0.0 --local-port=53
+    env_file: .env
+    ports:
+      - "53:53"
+      - "53:53/udp"
+      - "8082:8082" # HTTP API for the recursor
```

Comment:
Would it be useful to also include an IPFS node? NGINX proxy (http-only)?

Reply:
We'll probably need an IPFS node when we do the append endpoint stuff. I guess we could just map the ports to the local copy / something in our nix-shell? 🤷‍♀️

What do we want the NGINX proxy for here? TLS?

Reply:
> We'll probably need an IPFS node when we do the append endpoint stuff. I guess we could just map the ports to the local copy / something in our nix-shell? 🤷‍♀️

Added a local ipfs node for simplicity

> What do we want the NGINX proxy for here? TLS?

Oh, it's a pain to do localhost TLS -  I was thinking more just to emulate the gateway experience

Reply:
> Oh, it's a pain to do localhost TLS - I was thinking more just to emulate the gateway experience

I guess that means we'd have to turn off HTTPS if we were developing core fission apps like the auth lobby/drive against a locally-running fission constellation, right?